### ISSUE TITLE
Criteria for aux:pass: 'get'

### ISSUE THREAD

**Issue by nschneid** [UniversalDependencies/docs#647]
Most of the English treeabanks use `aux:pass` for passive auxiliaries. Canonically this is a form of "be", but "get" can also act as a passive aux:

- He got hit by a truck ≈ He was hit by a truck

However, "get" as a light verb is thorny and can also have causative and inchoative usages where no agent-by-phrase is possible:

- I got ready/old.
- I got everybody excited/on track.

In EWT (at least) there is perhaps an overuse of `aux:pass` for non-passive uses of "get". Some that seem dubious to me:

- get + ADJ: lucky, fussy, stuck
- get + participle:
  * get involved
  * get married (the agent of 'marry' can be an officiator, and "We got married by a/the priest" ~sounds odd in contemporary English~ is attested, but to me sounds a bit odd)
  * get fucked up ('drunk')
  * get rid of
  * get caught in traffic

etc.

There are others that could potentially have a by-phrase, but seem more inchoative than passive:

- get certified in Delaware ('become certified' > 'be certified by (authority)')
- get scared

**Comment by amir-zeldes** [comment]
Again offering GUM for comparison - we treat the ADJ class as `xcomp`, by analogy to "become", so "get mad" = xcomp(get,mad):

https://corpling.uis.georgetown.edu/annis/#_q=ImdldCIgLiAibWFkIg&_c=R1VN&cl=5&cr=5&s=0&l=10

get + VVN is interpreted as a passive auxiliary, though we probably also have some errors:

https://corpling.uis.georgetown.edu/annis/#_q=ImdldCIgXz1fIHRva19mdW5jPSJhdXhwYXNzIg&_c=R1VN&cl=5&cr=5&s=0&l=10

Examples like "involved" are probably considered JJ in most contexts based on the PTB tagging guidelines (Santorini 1990), so they should also be `xcomp` IMO. They usually pass the JJ tests:

  * Get very involved (cf. *"get very caught")
  * Get more involved (*"get more caught")
  * ?Get uninvolved (*"get uncaught")

Not totally sure about the last one, but overall it looks like JJ to me.

**Comment by nschneid** [comment]
#206 concerns which of these "gets" should be tagged `AUX`

**Comment by manning** [comment]
I agree, these should be fixed and changed to `xcomp`, as you describe. Maybe they will be for the next release, since we're getting a bit more updating done!

**Comment by dan-zeman** [comment]
Has this bug been fixed?

**Comment by nschneid** [comment]
Not entirely, as it's painful to go through all the tokens of "get" to decide which are actually passive auxes. But I will look into it for EWT.

**Comment by amir-zeldes** [comment]
Get-passive should be properly distinguished in GUM, so one could also try using a GUM trained parser to find likely errors.

**Comment by nschneid** [comment]
"I just got done looking at the underlying contract language": I'd say "get done" is an idiom meaning 'finish', but to analyze it compositionally would we say "looking at..." is an optional depictive advcl???

In general, looking at https://universal.grew.fr/?custom=655e761777b12, I think some are clear get-passives and others are borderline. @amir-zeldes maybe we should sit down and go through them sometime.

**Comment by amir-zeldes** [comment]
Sure, happy to talk about it. But as for this "done", I don't think it's a passive participle, since you can say "I'm done" using this kind of done, but you can't say "I'm done (*by them)". In GUM this reading of 'done' is tagged JJ and lemmatized 'done':

https://gucorpling.org/annis/#_q=bGVtbWE9ImRvbmUi&_c=R1VN&cl=5&cr=5&s=0&l=10

**Comment by nschneid** [comment]
Happy to change "done" to ADJ, but is advcl correct for the "looking" clause? Should it be a second xcomp?
